src/w3c/templates/headers.js

```diff
@@ -27,7 +27,7 @@ function getSpecTitleElem(conf) {
     `;
     specTitleElem.prepend(...childNodes);
   }
-  conf.title = specTitleElem.textContent.trim();
+  conf.title = specTitleElem.textContent.replace(/\s{2,}/g, " ").trim();
```

Comment:
Probably want `/\s\s+/g` here, no? Just normalizing whitespace?  

```suggestion
  conf.title = specTitleElem.textContent.replace(/\s{2,}/g, " ").trim();
```

Reply:
We have a `norm` in utils.js

Reply:
even better!

Reply:
@marcoscaceres Sorry, what's the difference between `/\s{2,}` and `/\s\s+`? 
@sidvishnoi Yes, we can use that. Thanks.

Reply:
> Sorry, what's the difference between /\s{2,} and /\s\s+?
I guess they are the same... just more used to seeing the other.